### PR TITLE
[MM-29515] Hide imports tab when in cloud installation

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
We don't want to show the old slack import tool in cloud installations. This PR will hide it from the menu if the instance is a cloud one.
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-29515
#### Related Pull Requests
https://github.com/mattermost/mattermost-server/pull/16271

#### Screenshots
N/A

### CODE REVIEW

**Review by stevemudie — APPROVED** [review]
Worked as expected.

File: package-lock.json [hunk 1]
```diff
@@ -6995,7 +6995,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```
Comment (marianunez) [inline package-lock.json:6998]: Are these changes intended?

**Review by marianunez — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (nickmisasi) [reply package-lock.json]: Nope! Fixed.

File: components/team_settings_modal/team_settings_modal.tsx [hunk 2]
```diff
@@ -13,7 +13,8 @@ const SettingsSidebar = React.lazy(() => import('components/settings_sidebar'));
 import TeamSettings from 'components/team_settings';
 
 type Props = {
-    onHide: () => void
+    onHide: () => void,
+    isCloud: boolean,
```
Comment (ethervoid) [inline components/team_settings_modal/team_settings_modal.tsx:17]: What about using `isCloud?: boolean` so we don't need to pass the flag always and just when we want?

**Review by ethervoid — APPROVED** [review]
LGTM, just a small comment
